user_guides/jakartaee/src/main/jbake/content/rules.adoc

```diff
@@ -459,12 +459,19 @@ TCK and the following Technology Compatibility Kits (TCKs):
 * Jakarta Dependency Injection 2.0 
 * Jakarta Bean Validation 3.0
 * Jakarta XML Binding 3.0 (If XML Binding is supported)
+* Jakarta JSON Processing 2.1
+* Jakarta JSON Binding 3.0
 
 In addition to the compatibility rules outlined in this TCK User's
 Guide, Jakarta EE 9.1 implementations must also adhere to all of the
 compatibility rules defined in the User's Guides of the aforementioned
 TCKs.
 
+EE21.1 If the Jakarta EE 10 implementation uses a runtime which has
+already been validated by the Technology Compatibility Kit,
+the Jakarta EE 10 implementation may use result of such validation
+to claim its compliance with the Technology Compatibility Kit.
+
```

Comment:
Please see above feedback on `EE-WP21.1` which I think also applies to `EE21.1`.